src-11ty/_includes/base.html

```diff
@@ -163,6 +163,11 @@
       .highlight + .card-content {
         margin-top: 16px;
       }
+      _::-webkit-full-page-media,
+      _:future,
```

Comment:
whut is this?! 😄

Reply:
I don't know. I just copied Stack overflow.